### PR TITLE
Support Azure Migrate Assessment Project (s)

### PR DESCRIPTION
**Pull request by sathish-progress**
### Description
Support Azure Migrate Assessment Project (s)

### Check List

- [X] New functionality includes tests
- [X] All tests pass
- [X] `rake lint` passes
- [X] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: docs/resources/azure_migrate_assessment_projects.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+| tags                           | A list of Tags provided by Azure Tagging service.                      | `tags`           |
+| properties                     | A list of Properties for all the projects.                             | `properties`     |
+| assessmentSolutionIds          | Assessment solution ARM ids tracked by `Microsoft.Migrate/migrateProjects`.| `assessmentSolutionId`  |
+| createdTimestamps              | Times when this project was created. Date-Time represented in ISO-8601 format.| `createdTimestamp`|
+| customerStorageAccountArmIds   | The ARM ids of the storage account used for interactions when public access is disabled.| `customerStorageAccountArmId` |
+| customerWorkspaceIds           | The ARM ids of service map workspace created by customer.              | `customerWorkspaceId` |
+| customerWorkspaceLocations     | Locations of service map workspace created by customer.                | `customerWorkspaceLocation`|
+| lastAssessmentTimestamps       | Storage Redundancy types offered by Azure.                             | `lastAssessmentTimestamp` |
```
Comment (sa-progress) [inline docs/resources/azure_migrate_assessment_projects.md:55]: Seems Description is wrong.Please check.

**Review by sa-progress — CHANGES_REQUESTED** [review]

**Review by sa-progress — APPROVED** [review]

File: docs/resources/azure_migrate_assessment_project.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+| location                      | Azure location in which project is created.                      |
+| properties.assessmentSolutionId | Assessment solution ARM id tracked by `Microsoft.Migrate/migrateProjects`.|
+| properties.customerStorageAccountArmId| The ARM ID of the storage account used for interactions when public access is disabled.|
+| properties.privateEndpointConnections | The list of private endpoint connections to the project. |
+| properties.numberOfMachines   | Number of machines in the project.                               |
+| tags                          | Tags provided by Azure Tagging service.                          |
+
+For properties applicable to all resources, such as `type`, `name`, `id`, and `properties`, refer to [`azure_generic_resource`](azure_generic_resource.md#properties).
+
+Also, refer to [Azure documentation](https://docs.microsoft.com/en-us/rest/api/migrate/assessment/projects/get) for other properties available. Any attribute in the response may be accessed with the key names separated by dots (`.`).
```
Comment (IanMadd) [inline docs/resources/azure_migrate_assessment_project.md:70]:
```suggestion
Refer to the [Azure documentation](https://docs.microsoft.com/en-us/rest/api/migrate/assessment/projects/get) for other properties available. Access any attribute in the response by separating the key names with a period (`.`).
```

File: docs/resources/azure_migrate_assessment_project.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+## Examples
+
+### Test that the Migrate Assessment Project has a minimum scalingFactor
```
Comment (IanMadd) [inline docs/resources/azure_migrate_assessment_project.md:74]:
```suggestion
### Test that the Migrate assessment project has a minimum scaling factor
```

**Review by IanMadd — COMMENTED** [review]
Feedback for @dkumaras

**Review by IanMadd — APPROVED** [review]

**Comment by sathish-progress** [comment]
@dkumaras , Could you please amend your signature to your commits that is causing the DCO to fail ?

**Pull request merged**